apps/cowswap-frontend/src/api/cowProtocol/errors/OperatorError.ts

```diff
@@ -50,12 +50,14 @@ export enum ApiErrorCodes {
   UNHANDLED_DELETE_ERROR = 'UNHANDLED_DELETE_ERROR',
 }
 
+const DuplicateOrderError = 'There was another identical order already submitted. Please try again.'
 export enum ApiErrorCodeDetails {
   AlreadyCancelled = 'Order is already cancelled.',
   AmountIsZero = 'Cannot place order with 0 amounts.',
   AppDataHashMismatch = 'AppDataHashMismatch',
   AppdataFromMismatch = 'AppdataFromMismatch',
-  DuplicateOrder = 'There was another identical order already submitted. Please try again.',
+  DuplicateOrder = DuplicateOrderError,
+  DuplicatedOrder = DuplicateOrderError,
```

Comment:
This should have been fixed already https://github.com/cowprotocol/services/issues/1928

```suggestion
  DuplicatedOrder = 'There was another identical order already submitted. Please try again.',
```